docs/styling/all-style-and-layout-props.md

```diff
@@ -0,0 +1,225 @@
+        "values": ["start", "end", "center", "justify", "left", "right"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/text-align",
+    },
+    "text_wrap": {
+        "description": "controls how text inside an element is wrapped",
+        "values": ["wrap", "nowrap", "balance", "pretty"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/text-wrap",
+    },
+
+    "top / bottom / right / left": {
+        "description": "sets the vertical / horizontal position of a positioned element. It does not effect non-positioned elements.",
+        "values": ["0", "4em", "10%", "20px"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/top",
+    },
+    "width": {
+        "description": "sets an element's width",
+        "values": ["150px", "20em", "75%", "auto"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/width",
+    },
+    "white_space": {
+        "description": "sets how white space inside an element is handled",
+        "values": ["normal", "nowrap", "pre", "break-spaces"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/white-space",
+    },
+    "word_break": {
+        "description": "sets whether line breaks appear wherever the text would otherwise overflow its content box",
+        "values": ["normal", "break-all", "keep-all", "break-word"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/word-break",
+    },
+    "z_index": {
+        "description": "sets the z-order of a positioned element and its descendants or flex and grid items, and overlapping elements with a larger z-index cover those with a smaller one",
+        "values": ["auto", "1", "5", "200"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/z-index",
+    },
+    
+
+}
+
+#  28. whitespace, 29. word-break
+
+
+def show_props(key, props_dict):
+    prop_details = props_dict[key]
+    return rx.table.row(
+        rx.table.cell(rx.code(key), justify="center",),
+        rx.table.cell(prop_details["description"], justify="center",),
+        rx.table.cell(rx.hstack(*[rx.badge(value) for value in prop_details["values"]], flex_wrap="wrap"), justify="center",),
+        rx.table.cell(rx.link(rx.icon("square_arrow_out_up_right", color=rx.color("mauve", 11), size=18,), href=prop_details["link"], is_external=True), justify="center",),
+        align="center",
+    )
+
+```
+
+
+```python eval
+rx.table.root(
+    rx.table.header(
+        rx.table.row(
+            rx.table.column_header_cell(
+                "Prop", justify="center"
+            ),
+            rx.table.column_header_cell(
+                "Description",
+                justify="center",
+                
+            ),
+            rx.table.column_header_cell(
+                "Potential Values",
+                justify="center",
+            ),
+            rx.table.column_header_cell(
```

Comment:
This column should be centered on the x-axis, align_items="center"

Reply:
changed